dask/optimize.py

```diff
@@ -45,7 +47,13 @@ def fuse(dsk):
     >>> d = {'a': 1, 'b': (inc, 'a'), 'c': (inc, 'b')}
     >>> fuse(d)  # doctest: +SKIP
     {'c': (inc, (inc, 1))}
+    >>> fuse(d, keys=['b'])  # doctest: +SKIP
+    {'b': (inc, 1), 'c': (inc, 'b')}
     """
+    if isinstance(keys, (set, list)):
+        keys = set(keys)
+    elif keys is not None:
+        keys = set([keys])
```

Comment:
I actually can't ever see passing in a single key to something called `keys`, so perhaps just assume an iterable, and always pass keys to set? Could even default to `()`, so there wouldn't even be a branch for `None`. Idk.